index.html

```diff
@@ -39,5 +40,144 @@ <h2>LET'S CONNECT</h2>
         </ul>
         
     </section>
-</body>
-</html>
\ No newline at end of file
+    <section class="project-container">
+        <div class="project">   
+        <img src="SnapshootPortfolio.png" alt="picture identity of a lady">
+        <h2>Tonic</h2>
+        <ol>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+            </ol>
+        <p>A daily selection of privately<br>
+             personalised reads: no accouts or <br>
+              sign-ups required. </p><br>
+    <ul>
+        <li>html</li>
+        <li>css</li>
+        <li>javascrpt</li>
+        </ul>
+        <button type="button" class="button"> See project</button>
+    </div>
+
+    
+    <div class="project">
+        <img src="SnapshootPortfolio1.png" alt="picture identity of a lady">
+        <h2>Multi-Post<br>Stories</h2>
+        
+        <ol>   
+        <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+        </ol>
+
+        <p>A daily selection of privately<br>
+            personalised reads: no accounts or <br>
+            sign-ups required.</p><br>
+            <ul>
+                <li>html</li>
+                <li>css</li>
+                <li>javascrpt</li>
+                 </ul>
```

Comment:
this part of you cards project doesn't look as the one in Figma design ,I encourage you to try stick to Figma design as much as possible 

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/166077473-3e75125c-f76e-4632-8774-bbc9c1c05d46.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/166077491-76719426-d05c-43fa-af0c-59b57c08c4ec.png)

Reply:
corrected  this too